src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -1326,6 +1323,173 @@ void S6Exporter::ExportMapAnimations()
     }
 }
 
+void S6Exporter::ExportTileElements()
+{
+    for (uint32_t index = 0; index < RCT2_MAX_TILE_ELEMENTS; index++)
+    {
+        auto src = &gTileElements[index];
+        auto dst = &_s6.tile_elements[index];
+        if (src->base_height == 0xFF)
+        {
+            std::memcpy(dst, src, sizeof(*src));
```

Comment:
What is this doing? Won't this be too big?

Reply:
It copies "null" tile elements over (mirroring S6Importer). It was indeed too big - I have corrected that now